src/FsAutoComplete.Core/SymbolLocation.fs

```diff
@@ -46,7 +45,7 @@ let getDeclarationLocation
         else
           loc.FileName
 
-      let taggedFilePath = UMX.tag normalizedPath
+      let taggedFilePath = Utils.normalizePath normalizedPath
```

Comment:
should we just remove `normalizedPath` directly above this and go right to `Utils.normalizePath`?

Reply:
Good catch